### PR TITLE
FAQ - update 'cause2001' entry

### PR DESCRIPTION
**Pull request by heinezen**
Changelog:
- Format list of instructions better

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -94,8 +94,8 @@
                             "The <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b> security certificate may be disabled on your device. This is a known cause to trigger the notification:",
                             "<b>'CAUSE: 2001 Something went wrong. An error occurred while trying to establish a secure connection to the server.'</b>",
                             "Follow these steps in order to activate the security certificate (The exact path depends on the device):",
```
Comment (MikeMcC399) [inline src/data/faq.json:96]: The punctuation is a bit strange:
"Follow these steps in order to activate the security certificate (The exact path depends on the device):",

Perhaps:
"Follow these steps in order to activate the security certificate:",
"(The exact path depends on the device.)",  ?

File: src/data/faq.json [hunk 2]
```diff
@@ -94,8 +94,8 @@
...
-                            "<ul><li>Settings > Biometrics and security > Other security settings > View security certificates, SYSTEM</li><li>Scroll down to <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b></li><li>Enable the certificate using the slider</li></ul>",
-                            "Should this procedure not solve this issue, please write a report here:  <a href='https://github.com/corona-warn-app/cwa-app-android/issues/968' target='_blank' rel='noopener'>cwa-app-android/issues/968</a>."
+                            "<ol><li>'Settings' -> 'Biometrics and security' -> 'Other security settings' -> 'View security certificates' -> 'SYSTEM'</li><li>Scroll down to <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b></li><li>Enable the certificate using the slider</li></ol>",
+                            "Should this procedure not solve this issue, please write a comment here:  <a href='https://github.com/corona-warn-app/cwa-app-android/issues/968' target='_blank' rel='noopener'>cwa-app-android/issues/968</a>."
```
Comment (MikeMcC399) [inline src/data/faq.json:98]: https://github.com/corona-warn-app/cwa-app-android/issues/968 is closed.

Some people might find that confusing.

File: src/data/faq_de.json [hunk 3]
```diff
@@ -93,8 +93,8 @@
                             "Das Sicherheitszertifikat <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b> ist möglicherweise auf Ihrem Gerät deaktiviert. Dies ist eine bekannte Ursache, um folgende Benachrichtigung auszulösen:",
                             "<b>'URSACHE: 2001 Etwas ist schiefgelaufen. An error occurred while trying to establish a secure connection to the server.'</b>",
                             "Führen Sie die folgenden Schritte aus, um das Sicherheitszertifikat zu aktivieren (Der genaue Pfad hängt vom Gerät ab).",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:95]: Suggest possible change from:

"Führen Sie die folgenden Schritte aus, um das Sicherheitszertifikat zu aktivieren (Der genaue Pfad hängt vom Gerät ab).",

to:

"Führen Sie die folgenden Schritte aus, um das Sicherheitszertifikat zu aktivieren:",
"(Der genaue Pfad hängt vom Gerät ab).",

File: src/data/faq_de.json [hunk 4]
```diff
@@ -93,8 +93,8 @@
...
-                            "Sollten diese Schritte das Problem nicht beheben, schreiben Sie bitte Ihren Bericht hier: <a href='https://github.com/corona-warn-app/cwa-app-android/issues/968' target='_blank' rel='noopener'>cwa-app-android/issues/968</a>."
+                            "<ol><li>'Einstellungen' -> 'Biometrische Daten und Sicherheit' -> 'Andere Sicherheitseinstellungen' -> 'Sicherheitszertifikate anzeigen' -> 'SYSTEM'</li><li>Scrollen Sie nach unten zu <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b></li><li>Aktivieren Sie das Zertifikat mit dem Schieberegler</li></ol>",
+                            "Sollten diese Schritte das Problem nicht beheben, schreiben Sie bitte einen Kommentar hier: <a href='https://github.com/corona-warn-app/cwa-app-android/issues/968' target='_blank' rel='noopener'>cwa-app-android/issues/968</a>."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:97]: https://github.com/corona-warn-app/cwa-app-android/issues/968 is already closed.

**Review by MikeMcC399 — COMMENTED** [review]
Please check the comments and decide if you want to make any changes.

File: src/data/faq.json (hunk 1 above)
Comment (heinezen) [reply src/data/faq.json]: fixed by removing the paranthesis. it's just two sentences now.

File: src/data/faq_de.json (hunk 3 above)
Comment (heinezen) [reply src/data/faq_de.json]: see above

File: src/data/faq.json (hunk 2 above)
Comment (heinezen) [reply src/data/faq.json]: I moved the whole entry to resolved.

File: src/data/faq_de.json (hunk 4 above)
Comment (heinezen) [reply src/data/faq_de.json]: see above

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
@heinezen @MikeMcC399 Thanks!

**Pull request merged**